googletest/test/gtest_list_tests_unittest.py

```diff
@@ -71,7 +71,7 @@
 TypedTest/0\.  # TypeParam = (VeryLo{245}|class VeryLo{239})\.\.\.
   TestA
   TestB
-TypedTest/1\.  # TypeParam = int\s*\*
+TypedTest/1\.  # TypeParam = int\s*\*( __ptr64)?
```

Comment:
This commit disappear after you merge PR #723. So ignore this commit here.
Specifically, these changes correct VC++ 64-bit tests